src/components/dropdowns/_dropdowns.scss

```diff
@@ -0,0 +1,105 @@
+// your components variables
+$dropdownFontSize: 12px;
+$dropdownHeight: 32px;
+$dropdownBorderSize: 2px;
+
+$include-html: false !default;
+
+@if ($include-html) {
+
+  .mint-dropdown {
+    @include component;
+    @extend .mint-icon-arrow_down;
+    
+    background-color: $grayAltLight;
+    border: $dropdownBorderSize solid $grayAltLight;
+    border-radius: 16px;
+    min-height: $dropdownHeight;
+    height: $dropdownHeight;
+    cursor: pointer;
+    @include hole;
+
+    &:hover,
+    &:active {
+      background-color: $graySecondary;
+      border-color: $graySecondary;
+    }
+
+    &:before {
+      $iconSize: $dropdownFontSize + $iconBoostValue;
+      font-size: $iconSize;
+      font-weight: bold;
+      position: absolute;
+      top: $dropdownHeight/4 - 2;
+      right: $dropdownHeight / 2 - $iconSize / 2;
+      color: $grayAlt;
+      z-index: 1;
+      transition: transform 0.3s;
+    }
+
+    &--full {
+      width: 100%;
+    }
+
+    &__hole {
+      justify-content: flex-start;
+      min-height: $dropdownHeight - (2 * $dropdownBorderSize);
+      margin: 0 $dropdownHeight 0 $dropdownHeight/2;
+      
+      &--active {
+        display: none;
+      }
+    }
+
+    &__items {
+      visibility: hidden;
+      padding: 0;
+      margin: 0 0 $layoutDefaultPadding/2;
+    }
+
+    &__item-hole {
+      display: flex;
+      align-items: center;
+      justify-content: flex-start;
+      height: 100%;
```

Comment:
check it now, please, previous screenshot (do not know why - was broken), i just updated it.